stm/stm.ml

```diff
@@ -2639,8 +2630,7 @@ let process_transaction ~doc ?(newtip=Stateid.fresh ()) x c =
            |> Exninfo.iraise
          else
 
-          let proof_mode = Some (Vernacinterp.get_default_proof_mode ()) in
-          let id = VCS.new_node ~id:newtip proof_mode () in
+          let id = VCS.new_node ~id:newtip () in
           let bname = VCS.mk_branch_name x in
```

Comment:
Yeah so if we don't store the state here, it means that we can't parse a proof now without executing the definition statement, right?

Reply:
I did update last commit, the STM should be behaving as before